demosplan/DemosPlanCoreBundle/Command/Data/GenerateCustomerCommand.php

```diff
@@ -86,6 +100,13 @@ protected function configure(): void
             InputOption::VALUE_REQUIRED,
             'The subdomain of the customer to be created. If omitted it will be asked interactively.'
         );
+        $this->addOption(
```

Comment:
i would set the default value as default, so that when nothing is given the default values are set

Reply:
I wanted to allow the option to insert empty strings as well.
But you can call the command vie ```-m default``` to use those defaults without any further interaction

Reply:
I see, fair point, but is this possible when the values are validated like this? (had no opportunity to check it myself)